DFE-Digital/Dfe.Academies.External
# amending API model properties to be correct types to fix deserilization

After last deployment, following error occurred in sentry:-
https://sentry.io/organizations/sdd-n7/issues/3544196368/?project=6047969&query=is%3Aunresolved

This is an error within the web application trying to unpack the get application response on previousschoolfinancialyear.revenuestatus

However, after fixing this, several other api model properties were incorrect

## Type of change

## Expected behaviour
Try and open land and buildings page within web application. nothing will be returned

## Steps to reproduce issue (if relevant)
1. Try and open land and buildings page within web application. nothing will be returned

## Steps to test this PR
1. run unit tests - CI / CD will run these automatically
2. Open web application and go into an existing application - data should be returned

## Prerequisites
n/a
